js/components/form.js

```diff
@@ -6,7 +6,18 @@ export const renderUserProfile = async () => {
     if(!hasUserData(myData)) return;
     
     const mainContent = document.getElementById('root');
-    let suffixList = {612166858: 0,255907182: 1,226924545: 2,270793412: 3,959021713: 4,643664527: 5,537892528: 6};
+    let suffixList = { 612166858: 0, 
```

Comment:
Would these keys make more sense being defined in `js/fieldToConceptIdMapping.js` and referenced here by name for readability? I know this is largely a whitespace change with a few additional keys, so it's fair if you would prefer not to do that at this time.